doc/specs/#607 - Commandline Arguments for the Windows Terminal.md

```diff
@@ -0,0 +1,729 @@
+  creating the window, and ignored for subsequent `new-tab` commands. When
+  combined with any of `--maximized` or `--fullscreen`, an error message will be
+  displayed to the user, indicating that an invalid combination of arguments was
+  provided.
+* `--initialRows rows`: Create the terminal window with `rows` rows (in
+  characters). If omitted, uses the value from the user's settings. This
+  parameter is only used when initially creating the window, and ignored for
+  subsequent `new-tab` commands. When combined with any of `--maximized` or
+  `--fullscreen`, an error message will be displayed to the user, indicating
+  that an invalid combination of arguments was provided.
+* `--initialCols cols`: Create the terminal window with `cols` cols (in
+  characters). If omitted, uses the value from the user's settings. This
+  parameter is only used when initially creating the window, and ignored for
+  subsequent `new-tab` commands. When combined with any of `--maximized` or
+  `--fullscreen`, an error message will be displayed to the user, indicating
+  that an invalid combination of arguments was provided.
+* `[terminal_parameters]`: See [[terminal_parameters]](#terminal_parameters).
+
+
+#### `split-pane`
+
+`split-pane [--target,-t target-pane] [-h]|[-v] [--percent,-% split-percentage] [terminal_parameters]`
+
+Creates a new pane in the currently focused tab by splitting the given pane
+vertically or horizontally.
+
+**Parameters**:
+* `--target,-t target-pane`: Creates a new split in the given `target-pane`.
+  Each pane has a unique index (per-tab) which can be used to identify them.
+  These indicies are assigned in the order the panes were created. If omitted,
+  defaults to the index of the currently focused pane.
+* `-h`, `-v`: Used to indicate which direction to split the pane. `-v` is
+  "vertically" (think `[|]`), and `-h` is "horizontally" (think `[-]`). If
+  omitted, defaults to vertical. If both `-h` and `-v` are provided, defaults to
+  vertical.
+* `--percent,-% split-percentage`: Designates the amount of space that the new
+  pane should take as a percentage of the parent's space. If omitted, the pane
+  will take 50% by default.
+* `[terminal_parameters]`: See [[terminal_parameters]](#terminal_parameters).
+
+#### `focus-tab`
+
+`focus-tab [--target,-t tab-index]`
+
+Moves focus to a given tab.
+
+**Parameters**:
+
+* `--target,-t tab-index`: moves focus to the tab at index `tab-index`. If omitted,
+  defaults to `0` (the first tab).
+
+#### `focus-pane`
+
+`focus-pane [--target,-t target-pane]`
+
+Moves focus within the currently focused tab to a given pane.
+
+**Parameters**:
+
+* `--target,-t target-pane`: moves focus to the given `target-pane`. Each pane
+  has a unique index (per-tab) which can be used to identify them. These
+  indicies are assigned in the order the panes were created. If omitted,
+  defaults to the index of the currently focused pane (which is effectively a
+  no-op)

```

Comment:
Consider adding a period at the end. This makes it consistent with other paragraphs.